org.lflang/src/org/lflang/validation/ValidatorErrorReporter.java

```diff
@@ -115,6 +115,20 @@ public String reportWarning(String message) {
         return message;
     }
 
+    /**
+     * Report the given message as a warning on the object currently under
```

Comment:
Incorrect comment. Just leave it out because it is an override, anyway?

Reply:
comment is removed.